doc/manual.rst

```diff
@@ -1282,6 +1282,19 @@ string from a cstring:
   var cstr: cstring = str
   var newstr: string = $cstr
 
+In C backend You shouldn't modify cstring literals. Otherwise your program will crash.
```

Comment:
This is our manual. We don't use "You" here. "will crash" is also too informal language.